IH1RZJ/model/dto/json/Appearance.cs

```diff
@@ -0,0 +1,12 @@
+using System.Text.Json.Serialization;
+
+namespace IH1RZJ.Model.DTO.Json;
+
+public record class AppearanceJsonDTO
+{
+  public Guid ID { get; init; } = Guid.NewGuid();
```

Comment:
.NET 9 óta van V7-es GUID is. Guid.CreateV7() asszem a factory metódusa. Amíg JSON-ba vagy teljesen jó a V4-is, de amint ezt DB-be akarod tenni, már nem. Órán majd kérdezz rá, hogy miért inkább V7-es Guid, ha érdekel.